public_src/components/modal/modal.component.ts

```diff
@@ -19,19 +19,32 @@ import * as L from 'leaflet';
 })
 
 export class ModalComponent {
+
+  public suggestedNames: string[];
+  public error: string;
+
+  @ViewChildren('chosenRef') chosenRefS ;
+  @ViewChildren('newlyAddedValue') newlyAddedValue ;
+
+  public errorObject: any;
```

Comment:
is it possible to use/create few new interfaces to describe what is actually added here later?